sites/public/src/forms/filters/FilterForm.tsx

```diff
@@ -152,6 +153,12 @@ const FilterForm = (props: FilterFormProps) => {
             }}
           />
         </div>
+        <label className="field-label">{t("listingFilters.region")}</label>
+        <div className="accessibility-filter-selector">
```

Comment:
I would separate out what css you want from this class, or if you want all of it, I would rename the class so its clear that this is being used more generally

Reply:
Renamed to checkbox-filter-group